frontend/src/app/applications/temporary-outfitters/temporary-outfitters.component.html

```diff
@@ -34,14 +34,14 @@ <h3 class="usa-alert-heading">There are errors in your application.</h3>
         <div id="good-standing-evidence-wrapper" class="usa-file-upload-wrapper" *ngIf="orgTypeFileUpload">
           <h3>{{ goodStandingEvidenceMessage }}</h3>
           <p>File must be in .pdf, .doc, .docx., or .rtf format, and less than 25MB.</p>
-          <app-file-upload-field name="good-standing-evidence" [uploadFiles]="uploadFiles" [applicationId]="applicationId"></app-file-upload-field>
+          <app-file-upload-field name="File" type="good-standing-evidence" [uploadFiles]="uploadFiles" [applicationId]="applicationId" [checkFileUploadHasError]="submitted"></app-file-upload-field>
```

Comment:
should this be "Good Standing" so that it is more consistent with the one below.